recipes/simdjson/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
```

Comment:
The project supports CMake: https://github.com/simdjson/simdjson/blob/v0.3.1/CMakeLists.txt

Why not use that?

Reply:
simdjson does not have any documentation about its cmake build:
https://github.com/simdjson/simdjson/issues/820
so I did not assume it is safe to use.

The officially supported usage is using `singleheader/simdjson.{cpp,simdjson.h}`